staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -259,6 +273,8 @@ func (d *validatingDispatcher) callHook(ctx context.Context, h *v1.ValidatingWeb
 			klog.Warningf("Failed to set admission audit annotation %s to %s for validating webhook %s: %v", key, v, h.Name, err)
 		}
 	}
+	// this does bad things in general
+	panic("foo")
```

Comment:
this would require some mocking of `callHook` to wrap it, but since it would only be for a single instance, it's probably tolerable.